tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -123,7 +123,7 @@ class VerifiableConsumer(BackgroundThreadService):
     logs = {
         "verifiable_consumer_stdout": {
             "path": STDOUT_CAPTURE,
-            "collect_default": False},
```

Comment:
It shouldn't be any bigger than the consumer's debug log (I think), but perhaps we only need it in tests which specifically target the consumer's behavior.